CHANGELOG.markdown

```diff
@@ -1,3 +1,10 @@
+naxt major
+----------
+
+* Add `Eq1`, `Ord1`, `Read1` and `Show1` instances for `Coyoneda` and `Yoneda`.
+* Change contexts of `Eq` and `Ord` instances of `Coyoneda` and `Yoneda` to require
```

Comment:
The `Read (f a) => Read (Coyoneda f a)` is because, `f` can be *GADT*, I guess you can still write `Read1 SomeGADT`, but I'm not sure it's worth. OTOH no reason not to change `Show (Coyoneda f a)`, as it *requires* `Functor f`. And dually for `Yoneda`.